app/views/admin/editions/_govspeak_help.html.erb

```diff
@@ -198,6 +198,11 @@
   $CTA</pre>
   </div>
 
+  <h3><a data-toggle="collapse" href="#brexit-cta" aria-expanded="false">Brexit call to action</a></h3>
```

Comment:
@vanitabarrett thinking about the stuff we had with the details component. Is there any other tracking stuff we'd expect to have bundled with this?

Reply:
Look at the other dropdowns, it doesn't look like we're currently tracking these links so it should be fine as is.

Reply:
Ace!